lib/utils/fs.go

```diff
@@ -291,6 +291,18 @@ func RemoveAllSecure(path string) error {
 // RemoveSecure attempts to securely delete the file by first overwriting the file with random data three times
 // followed by calling os.Remove(filePath).
 func RemoveSecure(filePath string) error {
+	info, err := os.Lstat(filePath)
+	if err != nil && os.IsNotExist(err) {
+		return nil
+	}
+	// don't fast return on other errors, still allow removeSecure to attempt removal
```

Comment:
```suggestion
	// Don't fast return on other errors, still allow removeSecure to attempt removal.
```

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
Happy to include this change.  However be aware there are thousands of comments in `teleport` that are not sentences (on a cursory search I found over 7000).  If this is a style we are adopting we should build a linter that enforces it.

Reply:
A linter might be too disruptive given the state of the codebase, but I do think the advice is worth following. Not all comments need to be sentences, but when they are there's no downside in following it, plus it's a good habit to build, regardless of the programming language.